src/Chainweb/Version.hs

```diff
@@ -858,6 +859,14 @@ enableSPVBridge Development = (>= 130)
 enableSPVBridge (FastTimedCPM g) = const $ g == pairChainGraph || g == petersonChainGraph
 enableSPVBridge _ = const True
 
+-- | Generate events for gas, coinbase, crosschain
+pactGenerateEvents :: ChainwebVersion -> BlockHeight -> Bool
+pactGenerateEvents Mainnet01 = (>= 1_600_000) -- 2021-05-07T14:14:46
+pactGenerateEvents Testnet04 = (>= 1_140_000) -- 2021-05-06T13:47:27
```

Comment:
👍 Looks good. These there the estimates that I get base on current block heights:

mainnet: 2021-05-07 14:37:06
testnet: 2021-05-06 13:54:06